apiserver/pkg/model/converter.go

```diff
@@ -446,12 +446,12 @@ func FromKubeToAPIComputeTemplates(configMaps []*corev1.ConfigMap) []*api.Comput
 func FromCrdToAPIJobs(jobs []*rayv1api.RayJob) []*api.RayJob {
 	apiJobs := make([]*api.RayJob, 0)
 	for _, job := range jobs {
-		apiJobs = append(apiJobs, FromCrdToApiJob(job))
+		apiJobs = append(apiJobs, FromCrdToAPIJob(job))
```

Comment:
I'm really surprised it's not updated in the last PR, my guess is linter pauses when it detects a number of errors.